### ISSUE TITLE
Batch  normalization - build in to layers?

### ISSUE THREAD

**Issue by AlexDBlack** [eclipse/deeplearning4j#4093]
Currently we have separate batch normalization layers.

It might be easier for users if we allowed something like "useBatchNormalization(true)" on each layer; currently, adding batch norm requires 3 layers (conv identity, batch norm, activation layer).

We already do this with dropout, for example.

**Comment by reuschling** [comment]
Would be nice if this also works with 1D CNN Layers then. Currently, batchNormalization as Layer with 1D CNN Layers seems to be not possible.
